sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java

```diff
@@ -235,6 +237,31 @@ private Object[][] gatewayAndDirect() {
         };
     }
 
+    @Test(groups = {"fast", "emulator"}, timeOut = TIMEOUT)
```

Comment:
FYI, this is still targeting "fast" group, please remove it, it will fail on live test pipeline.

Reply:
removed